frontend/actions/src/main/kotlin/com/virtuslab/gitmachete/frontend/actions/dialogs/FlatComboBoxUI.kt

```diff
@@ -0,0 +1,107 @@
+package com.virtuslab.gitmachete.frontend.actions.dialogs
+
+import com.intellij.ide.ui.laf.darcula.DarculaUIUtil
+import com.intellij.ide.ui.laf.darcula.ui.DarculaComboBoxUI
+import com.intellij.openapi.util.NlsContexts
+import com.intellij.ui.scale.JBUIScale
+import com.intellij.util.ui.ComponentWithEmptyText
+import com.intellij.util.ui.JBInsets
+import com.intellij.util.ui.JBUI
+import com.intellij.util.ui.StatusText
+import java.awt.Component
+import java.awt.Graphics2D
+import java.awt.Insets
+import java.awt.Rectangle
+import java.awt.geom.Line2D
+import java.awt.geom.Rectangle2D
+import java.awt.geom.RectangularShape
+import javax.swing.JButton
+import javax.swing.JComboBox
+import javax.swing.JComponent
+import javax.swing.JList
+import javax.swing.plaf.basic.ComboPopup
+
+/**
+ * ComboBox UI that enables to manager component side border and outer insets.
+ *
+ * @param border component border
+ * @param outerInsets component outer insets
+ * @param popupEmptyText text to show when component have no options
+ */
+internal class FlatComboBoxUI(
```

Comment:
Wow that's some brand new impl... or did you adapt it from somewhere?

Reply:
stolen from intellij as it's internal - but it turned out that we can relay on the base `DarculaComboBoxUI`

Reply:
Okay... but since it's `Darcula...`, will it also work in light theme?

Reply:
dARcula is not dRAcula

Reply:
<img width="1728" alt="Screenshot 2022-06-24 at 19 37 29" src="https://user-images.githubusercontent.com/19799111/176115429-f236831c-02db-45fa-8f31-9ee07435f463.png">
<img width="1728" alt="Screenshot 2022-06-24 at 19 37 29" src="https://user-images.githubusercontent.com/19799111/176115503-16648437-cc3a-4351-a607-220d3b1ff9fc.png">

the arrow is different (that's the only thing is see)

Reply:
light version:

<img width="464" alt="image" src="https://user-images.githubusercontent.com/19799111/176116542-43d815d5-cec1-44eb-95b7-5d95274b114e.png">

Reply:
![image](https://user-images.githubusercontent.com/3383210/176120996-dcda11b7-4b0a-46cc-a0c8-d4bcb9ebb6fd.png)

wow btw there's a separate build for Apple Silicon? 😯

Reply:
> > dARcula is not dRAcula
> 
> Hmmm? 🤔 as in, where's dRAcula mentioned?

I do not understand your question

Reply:
Why did you mention dRAcula in [your response](https://github.com/VirtusLab/git-machete-intellij-plugin/pull/944#discussion_r908072259)?